packages/core/src/abap/2_statements/expressions/string_template_formatting.ts

```diff
@@ -59,14 +59,17 @@ export class StringTemplateFormatting extends Expression {
                              str("ENGINEERING "),
                              new Source());
 
+    const decimalOptions = alt(new Integer(),
```

Comment:
Source includes Constant which includes Integer, so decimalOptions can be replaced with just Source